src/app/search-input/search-input.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="row no-gutters">
+	<div class="col-12">
+		<form (submit)="onSubmit()" class="form search-input">
+			<div class="form-group search-input__group">
+				<label for="search" class="search-input__label form-control mb-0">
+					<i class="fas fa-search"></i>
+					<input type="text"
+								 [formControl]="inputValue"
+								 class="form-control search-input__input"
+								 placeholder="search"
+								 id="search"
+								 (keyup)="onInputChange()"
```

Comment:
It will handle array keys too, those are useless in your case. Why don't you use `ngModel` and `ngModelChange`?

Reply:
fixed